### PR TITLE
update script.js

### PR DESCRIPTION
**Pull request by MarcoSteinke**
When adding 2 `reLU` activated layers which a relatively huge amount of units and subsequently decreasing the amount of units per layers, the approximation gets much closer to the real data. 

The following image shows this improvement

You can also increase the learning rate in the beginning of the training process by decreasing the batch size to "8". Then your model is more likely to really learn the data and you can prevent overfitting.

Now your model is already at a viable accurace after around 5 epochs, where its loss converges at around "0.0133".

The following image shows this result:

### CODE REVIEW

**Comment by MarcoSteinke** [comment]
I know this is just a tutorial project, I did this tutorial myself a year ago 😄 But I think I had to share my thoughts with you :)
